frontend/src/views/groupmember/CancelGroupMember.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div id="cancelGroupMember" v-if="inputFormActive">
+    <form @submit.prevent="submitForm">
+      <AppRow>
+        <AppCol class="col3">
+          <AppInput :e-model="v$.groupNumber" id="groupNumber" label="Group Number" type="text" v-model.trim="groupNumber" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col3">
+          <AppInput :e-model="v$.phn"  id="phn" label="Group Member's PHN" type="text" v-model="phn"/>
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col3">
+          <AppDateInput :e-model="v$.cancelDate" id="dp-input-cancelDate" label="Coverage Cancel Date" tooltip tooltipText="Date always defaults to last day of the month"
+                        monthPicker inputDateFormat="yyyy-MM" placeholder="YYYY-MM" v-model="cancelDate"/>
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppCol class="col3">
+          <AppSelect :e-model="v$.cancelReason" id="priorResidenceCode" label="Cancel Reason" v-model="cancelReason" :options="cancelReasons" />
+        </AppCol>
+      </AppRow>
+      <AppRow>
+        <AppButton :submitting="submitting" mode="primary" type="submit">Submit</AppButton>
+        <AppButton @click="resetForm" mode="secondary" type="button">Clear</AppButton>
+      </AppRow>
+    </form>
+  </div>
+  <div id="confirmation" v-if="!inputFormActive">
+    <br />
+    <p>PHN: {{ result?.phn }}</p>
+    <AppButton @click="resetForm" mode="primary" type="button">Cancel Another Group Member</AppButton>
+  </div>
+</template>
+
+<script>
+import useVuelidate from "@vuelidate/core";
+import { helpers, required } from "@vuelidate/validators";
+import { VALIDATE_GROUP_NUMBER_MESSAGE, VALIDATE_PHN_MESSAGE, validateGroupNumber, validatePHN } from "../../util/validators";
+import GroupMemberService from "../../services/GroupMemberService";
+
+export default {
+  name: 'CancelGroupMember',
+  setup() {
+    return {
+      v$: useVuelidate()
+    }
+  },
+  data() {
+    return {
+      inputFormActive: true,
+      submitting: false,
+      groupNumber:'',
+      phn: '',
+      cancelDate: null,
+      cancelReason: '',
+      result: {
+        phn: '',
+        status: '',
+        message: '',
+      },
+      cancelReasons: [
```

Comment:
We have used constants for other select options but I doubt these values will be re-used elsewhere.

Reply:
Will refactor if we need to reuse